extensions/common/json_util.cc

```diff
@@ -20,12 +20,13 @@
 namespace Wasm {
 namespace Common {
 
-::nlohmann::json JsonParse(absl::string_view str) {
-  return ::nlohmann::json::parse(str, nullptr, false);
+::nlohmann::json JsonParse(absl::string_view str, const bool allow_exceptions) {
```

Comment:
I think that we should move this utility to a common place. cc. @kyessenov

Reply:
This is a common place for istio extensions. Everything under extensions is self-contained.